demos/bartlett1932/experiment.py

```diff
@@ -96,13 +96,12 @@ class Bot(BotBase):
     def participate(self):
         """Finish reading and send text"""
         try:
-            time.sleep(5)
-            self.driver.execute_script("create_agent_failsafe")
             logger.info("Entering participate method")
+            self.driver.execute_script("create_agent")
```

Comment:
@cguardia Interesting. Can you help me understand why it's necessary to have the Selenium driver execute this particular script but not the others? What is it about the way that this experiment is written that prevents it from running without this kind of "reaching inside" the JavaScript?